spec/factories/invoices.rb

```diff
@@ -1,6 +1,11 @@
 FactoryBot.define do
   factory :invoice do
-    status { 1 }
-    customer { nil }
+    id { Faker::Number.unique.number }
```

Comment:
Looks like we missed this spot as a place that we can remove the id attribute from our factory bot since we figured out the create method will let it generate the id, created at, etc.

Reply:
We found another one and took it out, not needed after using the create_list instead of build_list.